packages/node/src/utils/cosmos.ts

```diff
@@ -169,7 +192,9 @@ function wrapCosmosMsg(
     block: block,
     msg: {
       typeUrl: rawMessage.typeUrl,
-      ...api.decodeMsg<any>(rawMessage),
+      get decodedMsg() {
```

Comment:
Is this changing the CosmosMessage type? Will this require updating other projects?

Reply:
Looks like that the data were originally accessed through msg property, will have to be accessed through msg.decodedMsg

Reply:
We could look into a way of transforming it so that the projects or dictionary remains unaffected